src/fragments/lib/auth/android/getting_started/30_initAuth.mdx

```diff
@@ -2,7 +2,7 @@ Add the Auth plugin before calling `Amplify.configure`. Update [the code you add
 
 <BlockSwitcher>
 <Block name="Java">
-  
+ 
```

Comment:
```suggestion

```
It looks like there are two spaces on this line. I still see the issue when removing only one space, but removing both spaces fixes the issue for me.